eosdis-nasa/earthdata-pub-dashboard
# EDPUB-1386: Create Step Add / Edit / List Pages

# Description

Adds dashboard pages for step add / edit / list

## Spec

See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1387

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull this branch and the corresponding API branch.
3. Navigate to http://localhost:3000/steps
4. Click the add button to add a step.
5. Click on edit button of the step you just added and test editing the step.

---
